compiler/quilt/nodes.py

```diff
@@ -44,6 +45,11 @@ def __init__(self, package, node, data=None):
     def __call__(self):
         return self._data()
 
+    @property
```

Comment:
Let's not bloat nodes.py with export-specific helpers. It'll be cleaner to just make a helper in export instead of adding methods here.

Reply:
Ah.  I thought that would be a useful thing for users, because it tells them whether or not it has a data file without actually loading (possibly gigabytes of) data.

Reply:
I'll remove that, though.